pip-freeze.txt

```diff
@@ -58,5 +58,6 @@ wheel==0.24.0
 xlrd==0.9.3
 xlutils==1.7.1
 xlwt==1.0.0
-supervisor==3.2
```

Comment:
the reason we were installing supervisor like this is because we need version 3.2 in order to pass env vars into the python process and the apt version is not 3.2